### ISSUE TITLE
Enhancing Home page

### ISSUE THREAD

**Issue by coding-geek21** [coding-geek21/classDeck#11]

**Comment by coding-geek21** [comment]
> _No description provided._

**Comment by yaswanthsaivendra** [comment]
@kaustav202 , You already have 2 assigned issues. Ask for assignment of a new one once you are done with them.

**Comment by coding-geek21** [comment]
> Can I take this up @coding-geek21

Complete Issue #2  first before jumping to this

**Comment by kaustav202** [comment]
Okay sure

**Comment by chinmayparekh** [comment]
I would like to work on it @coding-geek21 @yaswanthsaivendra

**Comment by nikitadhona** [comment]
Can I work on this?

**Comment by Kritinp** [comment]
Hey @coding-geek21 can you please assign this task to me?

**Comment by devanshpratapsingh** [comment]
@yaswanthsaivendra @coding-geek21 can you please assign me to this issue. I'll add interesting animations using CSS and try to beautify the home page more.

**Comment by Samridhi015** [comment]
@coding-geek21 assign me this issue, I'll work on enhancing it

**Comment by codervivek5** [comment]
i want to work on this issue. may you plz assign it me.

**Comment by coding-geek21** [comment]
@nikitadhona  Any updates?

**Comment by nikitadhona** [comment]
I am not familiar with Django. So i am unable to update stuffs :( . You can assign it to someone else,till i figure out @coding-geek21

**Comment by coding-geek21** [comment]
@Kritinp Assigning you. Make sure you complete it within 2 days

**Comment by Kritinp** [comment]
> @Kritinp Assigning you. Make sure you complete it within 2 days

Almost finished with it, also working on another issue.

**Comment by coding-geek21** [comment]
@Kritinp  any updates?

**Comment by codervivek5** [comment]
I want to work on this issue..plz assign me

**Comment by coding-geek21** [comment]
@Kritinp When are you gonna make PR? any updates?

**Comment by Kritinp** [comment]
> @Kritinp When are you gonna make PR? any updates?

I have fixed this issue by updating the existing PR.

**Comment by coding-geek21** [comment]
@Kritinp Did you enhance the home page?

**Comment by coding-geek21** [comment]
Anyone willing to work on this issue? comment so that i can assign you

**Comment by Kritinp** [comment]
> @Kritinp Did you enhance the home page?

Yeah, I added smooth scrolling and fixed some issues with mobile compatability. This was also included in the PR

**Comment by coding-geek21** [comment]
> > @Kritinp Did you enhance the home page?
> 
> Yeah, I added smooth scrolling and fixed some issues with mobile compatability. This was also included in the PR

Okay fine.. But anyway still the Home page can be improved by adding some js effects cards with more details about the website so I need someone to work on the homepage

**Comment by kaustav202** [comment]
Let me work on it @coding-geek21

**Comment by coding-geek21** [comment]
Go ahead @kaustav202

**Comment by coding-geek21** [comment]
@kaustav202 Any updates??

**Comment by kaustav202** [comment]
> @kaustav202 Any updates??

It'll be done by wed @coding-geek21

**Comment by coding-geek21** [comment]
Any updates @kaustav202

**Comment by dvlsatya** [comment]
I want to work on this issue. can you please assign me?

**Comment by coding-geek21** [comment]
@dvlsatya  any updates?

**Comment by nik132-eng** [comment]
if this issue is still open I want to work on this issue. can you please assign me this issue? for GSSoC

**Comment by Kanika2772002** [comment]
Hey @coding-geek21 can you please assign this task to me?

**Comment by Kanika2772002** [comment]
Ok

**Comment by karantec** [comment]
please assign me this issue under gSSOC 2023
